examples/preprocessing/eeg_bridging.py

```diff
@@ -0,0 +1,255 @@
+preprocessing. Therefore, the recommendation is to check for electrode
+bridging early in data collection and address the problem. Or, if the data
+has already been collected, quantify the extent of the bridging so as not
+to introduce bias into the data from this effect and exclude subjects with
+bridging that might effect the outcome of a study. Preventing electrode
+bridging is ideal but awareness of the problem at least will mitigate its
+potential as a confound to a study. This tutorial follows
+https://psychophysiology.cpmc.columbia.edu/software/eBridge/tutorial.html.
+"""
+# Authors: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD-3-Clause
+
+# %%
+
+# sphinx_gallery_thumbnail_number = 2
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+import mne
+
+print(__doc__)
+
+# %%
+# Compute Electrical Distance Metric
+# ----------------------------------
+# First, let's compute electrical distance metrics for a group of example
+# subjects from the EEGBCI dataset in order to estimate electrode bridging.
+# The electrical distance is just the variance of signals subtracted
+# pairwise. Channels with activity that mirror another channel nearly
+# exactly will have very low electrical distance. By inspecting the
+# distribution of electrical distances, we can look for pairwise distances
+# that are consistently near zero which are indicative of bridging.
+#
+# .. note:: It is likely to be sufficient to run this algorithm on a
+#           small portion (~3 minutes is probably plenty) of the data but
+#           that gel might settle over the course of a study causing more
+#           bridging so using the last segment of the data will
+#           give the most conservative estimate.
+
+ed_data = dict()  # electrical distance/bridging data
+raw_data = dict()  # store infos for electrode positions
+for sub in range(1, 11):
+    print(f'Computing electrode bridges for subject {sub}')
+    raw = mne.io.read_raw(mne.datasets.eegbci.load_data(
+        subject=sub, runs=(1,))[0], preload=True, verbose=False)
+    mne.datasets.eegbci.standardize(raw)  # set channel names
+    montage = mne.channels.make_standard_montage('standard_1005')
+    raw.set_montage(montage, verbose=False)
+    raw_data[sub] = raw
+    ed_data[sub] = mne.preprocessing.compute_bridged_electrodes(raw)
+
+
+# %%
+# Examine an Electrical Distance Matrix
+# -------------------------------------
+# Before we look at the electrical distance distributions across subjects,
+# let's look at the distance matrix for one subject and try and understand
+# how the algorithm works. We'll use subject 6 as it is a good example of
+# bridging. In the zoomed out color scale version on the right, we can see
+# that there is a distribution of electrical distances that are specific to
+# that subject's head physiology/geometry and brain activity during the
+# recording. On the right, when we zoom in, we can see several electrical
```

Comment:
I didn't get what the plot on the right shows at first and only looking through the code found the `vmax` limit. Maybe instead of "zoom" there should be a reference to "clipping" values above specific value?